vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -96,9 +96,9 @@ $global:imagesToPull += @(
     "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.30.0-windows-hp", # for k8s 1.29.x
     # Addon of Azure secrets store. Owner: jiashun0011 (Jiashun Liu)
     "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v1.3.4",
-    "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v1.4.0",
```

Comment:
Is v1.4.0 still used? Could we remove v1.3.4 and add v1.4.1?
Same for v1.5.1

Reply:
`1.4.0` was never used. For driver, we bumped version directly from `v1.3.4` to `v1.4.1` in RP code. Same as provider.